src/tiledb/cloud/groups.py

```diff
@@ -173,7 +177,9 @@ def deregister(
                     grp: rest_api.GroupInfo = m.group
                     deregister(grp.tiledb_uri, recursive=recursive)
                 else:
-                    raise tce.TileDBCloudError("unexpected group member type")
+                    raise tiledb_cloud_error.TileDBCloudError(
+                        "unexpected group member type"
+                    )
```

Comment:
I used functions in `tiledb.cloud.array` as a template and since they were using `tiledb_cloud_error` unaliased, I dealiased our existing code.